### PR TITLE
Add missing publicTransferBuild variant

### LINKED ISSUE #91: `publicTransfer` naming convention and missing build variant
Currently there's only a `publicTransfer` function in the manta-utilities class, which submits the transaction. 
It's not following the send-build convention of the rest of the extrinsic wrapper functions.
Furthermore the build variant is missing alltogether.

### PR DESCRIPTION
**Pull request by ghzlatarev**
closes #91 

* Rename publicTransfer to publicTransferSend to follow the established send/build convention
* Added the missing publicTransferBuild variant and example how to use it.

---

Before we can merge this PR, please make sure that all the following items have been checked off:

- [x] Linked to an issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Added **one** line describing your change in [`CHANGELOG.md`](https://github.com/manta-network/sdk/blob/main/CHANGELOG.md) and added the appropriate `L-` label to the PR.
- [x] Re-reviewed `Files changed` in the GitHub PR explorer.

### CODE REVIEW

File: manta-js/examples/asset-webpack-ts/index.ts [hunk 1]
```diff
@@ -62,6 +63,24 @@ const publicTransferTest = async () => {
...
+    const privateWalletConfig = {
+        environment: Environment.Production,
+        network: Network.Dolphin
+    }
+
+    const privateWallet = await MantaPrivateWallet.init(privateWalletConfig);
+
+    const destinationAddress = "5FHT5Rt1oeqAytX5KSn4ZZQdqN8oEa5Y81LZ5jadpk41bdoM";
+    const assetId = new BN("1"); // DOL
+    const amount = new BN("10000000000000000000"); // 10 units
```
Comment (ferrell-code) [inline manta-js/examples/asset-webpack-ts/index.ts:77]: 18 decimal points here instead of 12

Comment (ghzlatarev) [reply manta-js/examples/asset-webpack-ts/index.ts]: Yeah nice catch, the rest are fixed in the other PR.

**Review by ferrell-code — DISMISSED** [review]
I think a good clean up for example would be to have this in top level:
```javascript
const DOL = 1000000000000000
```

So big number is only written once. Separate issue from this PR tho

**Comment by ghzlatarev** [comment]
> 

https://github.com/Manta-Network/sdk/issues/97

File: manta-js/package/src/utils.ts [hunk 2]
```diff
@@ -63,4 +63,15 @@ export class MantaUtilities {
       console.error(e);
     }
   }
+
+  /// Creates a public transfer payload.
+  static async publicTransferBuild(api:ApiPromise, assetId: BN, amount: BN, destinationAddress: Address): Promise<any> {
+    const assetIdArray = Array.from(MantaPrivateWallet.assetIdToUInt8Array(assetId));
```
Comment (ferrell-code) [inline manta-js/package/src/utils.ts:69]: Removed this function in favor of this: https://github.com/Manta-Network/sdk/blob/37e7a879850639c95b93fe993ab02c2a1d145020/manta-js/package/src/privateWallet.ts#L375

**Review by ferrell-code — DISMISSED** [review]

File: manta-js/package/src/utils.ts [hunk 3]
```diff
@@ -47,8 +47,8 @@ export class MantaUtilities {
     }
   }
 
-  /// Executes a public transfer.
-  static async publicTransfer(api:ApiPromise, assetId: BN, amount: BN, destinationAddress: Address, senderAddress:Address, polkadotSigner:Signer): Promise<void> {
+  /// Builds, signs and sends a public transfer transaction.
+  static async publicTransferSend(api:ApiPromise, assetId: BN, amount: BN, destinationAddress: Address, senderAddress:Address, polkadotSigner:Signer): Promise<void> {
     api.setSigner(polkadotSigner);
     try {
       const assetIdArray = bnToU8a(assetId, {bitLength: 256});
```
Comment (ferrell-code) [inline manta-js/package/src/utils.ts:54]: repeats logic here should just call `publicTransferBuild`

File: manta-js/package/src/utils.ts [hunk 4]
```diff
@@ -63,4 +63,15 @@ export class MantaUtilities {
       console.error(e);
     }
   }
+
+  /// Creates a public transfer payload.
+  static async publicTransferBuild(api:ApiPromise, assetId: BN, amount: BN, destinationAddress: Address): Promise<any> {
```
Comment (ferrell-code) [inline manta-js/package/src/utils.ts:68]: possible to return a type here like `SubmitableExtrinsic` rather than `any`?

**Review by ferrell-code — APPROVED** [review]

**Review by Kevingislason — APPROVED** [review]

**Pull request merged**